packages/auth-providers/dbAuth/web/src/__tests__/dbAuth.test.ts

```diff
@@ -4,8 +4,10 @@ import { CurrentUser } from '@redwoodjs/auth'
 
 import { createDbAuthClient, DbAuthClientArgs, createAuth } from '../dbAuth'
 
-process.env.RWJS_API_URL = '/.redwood/functions'
-process.env.RWJS_API_GRAPHQL_URL = '/.redwood/functions/graphql'
+// @NOTE there are TS errors assigning these, because there's no ambient declarations
+// But because this is going to be moved as part of decoupling, ignoring for now
```

Comment:
> // But because this is going to be moved as part of decoupling, ignoring for now

Not decoupling auth right? Some other decoupling? Just clarifying

Reply:
Hey I've fixed this... I misread the path, so thought it wouldn't be needed!